README.md

```diff
@@ -19,15 +17,25 @@ To run the project:
         export DB_PASSWORD="communityfund_password"
 3. Run start
 
-        source .env && yarn start-dev-db
-        source .env && yarn start
+        yarn start:all
```

Comment:
We are already sourcing in tools/startDevDatabase.sh
Not sure about the way to go. Probably forcing it existence and baking it into the code is the right thing to do for now.